ts/modules/drag-panes.src.ts

```diff
@@ -250,32 +264,26 @@ var hasTouch = H.hasTouch,
              */
             y: 0
         }
-    };
-
-merge(true, Axis.prototype.defaultYAxisOptions, resizerOptions);
```

Comment:
That is missing. You should add something like `merge(true, Axis.defaultYAxisOptions, AxisResizer.resizerOptions);` at the end of the file.